README.md

```diff
@@ -27,6 +27,14 @@ in hand you are ready to start using twarc.
 1. install [Python](http://python.org/download) (2 or 3)
 2. pip install twarc (if upgrading: pip install --upgrade twarc)
 
+### Homebrew (macOS only)
+
+For macOS users, you can install `twarc` via:
+
+```bash
+$ brew install twarc
```

Comment:
Tip: use bash when there's no prompt and console when there is a prompt.

For example, bash with no prompt:

```bash
brew install twarc && example1
example2
```

console with prompt:

```console
$ brew install twarc && example1
here's some output
$ example2
more output
```